ui/app/AppLayouts/Profile/Sections/Ens/List.qml

```diff
@@ -7,6 +7,7 @@ import "../../../Chat/ChatColumn/MessageComponents"
 
 Item {
     property var onClick: function(){}
+    property var onSelectENS: function(){}
```

Comment:
Is this better than using a signal?
```
signal selectEns(string username)
```
Which can be used like:
```
List {
    onSelectEns: {
        console.log(“username:”, username)
    }
}
```

Reply:
I did not know that was possible